### DISCUSSION TITLE
Fill Tool

### DISCUSSION THREAD

**Discussion by Velkas** [danielebruneo/remarkable2-hacks#6] [Ideas]
With the additional tools and tool options, hacks offers a compelling reason to modify the vanilla RM2 system.

It would be nice to have a fill tool available that functions similarly to MS Paint-- Being able to fill an enclosed space with the selected color density. Additionally, being able to fill an entire layer if there are no closing bounds detected.

**Comment by dominiwe** [comment]
Might be best to suggest that in [ddvk/remarkable-hacks](https://github.com/ddvk/remarkable-hacks) directly. It seems that this is more of an informational repo, with guides and a list of such hacks :)

**Reply by danielebruneo** [reply]
that's correct.
We could still collect those as suggestions for ddvk, but we'll need to open an issue there at the end.

**Reply by Velkas** [reply]
Ah yes, I see. Thank you for explaining! I'll forward over that info directly. Feel free to close this one down otherwise I'll put up a reference link here when I get it posted.

**Comment by Velkas** [comment]
Looks like a ticket was already created over on [ddvk/remarkable-hacks](https://github.com/ddvk/remarkable-hacks) for primitive tools including fill. I've added my $0.02 there and am linking that discussion here.

https://github.com/ddvk/remarkable-hacks/issues/202
